WordPress/Classes/Networking/TaxonomyServiceRemoteXMLRPC.m

```diff
@@ -1,46 +1,132 @@
 #import "TaxonomyServiceRemoteXMLRPC.h"
 #import "RemotePostCategory.h"
 #import "RemotePostTag.h"
+#import "RemoteTaxonomyPaging.h"
 #import <WordPressShared/NSString+Util.h>
 #import <WordPressApi/WordPressApi.h>
 
+static NSString * const TaxonomyServiceRemoteXMLRPCCategoryTypeIdentifier = @"category";
+static NSString * const TaxonomyServiceRemoteXMLRPCTagTypeIdentifier = @"post_tag";
+
 @implementation TaxonomyServiceRemoteXMLRPC
 
 #pragma mark - categories
 
+- (void)createCategory:(RemotePostCategory *)category
+               success:(void (^)(RemotePostCategory *))success
+               failure:(void (^)(NSError *))failure
+{
+    NSMutableDictionary *extraParameters = [NSMutableDictionary dictionaryWithCapacity:2];
```

Comment:
Setting the capacity to 2 seems a bit weird when the dictionary could have a single key/value pair. Maybe just `[NSMutableDictionary dictionary]`?